cmd/common_test.go

```diff
@@ -128,6 +128,15 @@ func mustMarshalToMultiYAML[R any](t *testing.T, data []R) []byte {
 	return []byte(strings.Join(parts, "\n---\n"))
 }
 
+func mustJsonDeepCopy[O any](t *testing.T, object O) O {
```

Comment:
Prevents unwanted modification of original struct on `networkToUpdate := network1`.